core/spec/models/spree/calculator/flat_percent_item_total_spec.rb

```diff
@@ -4,6 +4,11 @@
   let(:calculator) { Spree::Calculator::FlatPercentItemTotal.new }
   let(:line_item) { mock_model Spree::LineItem }
 
+  describe ".description" do
+    subject{ described_class.description }
```

Comment:
There's a space missing here before the opening `{`. Can you add that one space? I know it's lots of commits to go and edit, but interactive rebase makes that pretty straightforward. Read up [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i) about it, it's great. And yes, please force push afterwards, we don't mind for open PRs before they're merged.